frontend/src/queries/schema.ts

```diff
@@ -283,7 +283,12 @@ export interface InsightsQueryBase extends Node {
     aggregation_group_type_index?: number
 }
 
-export type TrendsFilter = Omit<TrendsFilterType, keyof FilterType> // using everything except what it inherits from FilterType
+// using everything except what it inherits from FilterType
+// replace hidden_legend_keys with hidden_legend_indexes
```

Comment:
Using JSDoc comments `/** */` would enable IDEs to surface this, but that's a nit

Reply:
Ah, you're right. Will change this to JSDoc.